bower.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "ember-cli-github-pages",
   "dependencies": {
-    "handlebars": "~1.3.0",
-    "jquery": "^1.11.1",
-    "ember": "1.8.1",
-    "ember-data": "1.0.0-beta.12",
-    "ember-resolver": "~0.1.10",
-    "loader.js": "stefanpenner/loader.js#1.0.1",
-    "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
-    "ember-load-initializers": "stefanpenner/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0"
+    "ember": "2.0.1",
+    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
+    "ember-cli-test-loader": "0.2.0",
+    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
+    "ember-qunit": "0.4.10",
+    "ember-qunit-notifications": "0.0.7",
+    "jquery": "^1.11.3",
+    "loader.js": "3.3.0",
+    "qunit": "~1.18.0"
+  },
+  "resolutions": {
+    "ember": "2.0.1"
```

Comment:
I think https://github.com/emberjs/rfcs/pull/88 would help address surprise compat issues...also, an addon such as [Ember-Version-Is](https://github.com/hhff/ember-version-is) can easily be leveraged to allow addons to develop with the latest features while maintaining compatibility to older versions of Ember, much like we might have fallbacks for older browser versions while continuing to take advantage of newer specs supported by the latest releases. I don't think that addons should hold themselves back just for the sake of peace of mind, because the opposite problem occurs. If Addon X is on Ember 1.9.x, developers feel really squirmish about trusting it to work in something like the 2.x world. By setting the actual Ember version to the latest, you make it obvious that version is supported, which I think is the most important information.  Backwards compatibility can then be documented in the README as well as in the package.json (or however my RFC turns out).